### DISCUSSION TITLE
Experimental feature: show paths mode

### DISCUSSION THREAD

**Discussion by othmar52** [jellyfin/jellyfin-web#6353] [Ideas]
Currently, verifying whether media identification is accurate or problematic can be quite tedious.  
In my case, the Metadata Manager often proves ineffective, especially for users with large media collections.  
To address this, I implemented an optional feature to display file paths directly in the frontend, making it much easier to validate media identification.  

I think it is especially useful after a fresh jellyfin install.  

see an already working raw concept of this feature in #6351

**Comment by thornbill** [comment]
From the PR:

> I can this type of functionality being potentially useful as part of the metadata manager, but I would not accept it as part of the main user facing UI.
